cirq-superstaq/cirq_superstaq/compiler_output.py

```diff
@@ -146,6 +146,23 @@ def __repr__(self) -> str:
             f"{self.pulse_lists!r})"
         )
 
+    def __repr_pretty__(self) -> str:
```

Comment:
to be picked up by ipython/jupyter/etc i think this needs to be
```suggestion
    def _repr_pretty_(self, p, cycle=False) -> None:
        ...
        p.text(<string>)
```
(see https://ipython.readthedocs.io/en/stable/config/integrating.html#pretty-printing)